cypress/integration/submit_spec.js

```diff
@@ -29,7 +29,7 @@ describe('e2e test', () => {
     cy.get('#field-input--giftaid').click().click();
     cy.get('#field-error--giftaid>span').should('contain', 'To Gift Aid your donation you need to tick the checkbox');
     cy.get('#field-input--giftaid').click();
-    cy.get('#field-error--giftaid>span').should('be.not.visible')
+    cy.get('#field-error--giftaid>span').should('not.exist')
```

Comment:
In this newer version of Cypress, `'be.not.visible'` specifically relates to something still being in the DOM, but with a display:none rule; these error messages are dynamically added and removed, so just needed to have that reflected here.